pkg/operator2/deployment.go

```diff
@@ -47,6 +48,11 @@ func defaultDeployment(
 	}
 	deployment.Spec.Template.Annotations[deploymentVersionHashKey] = rvsHashStr
 
+	// Ensure a rollout when the bootstrap user goes away
+	if bootstrapUserExists {
+		deployment.Spec.Template.Annotations["operator.openshift.io/bootstrap-user-exists"] = "true"
```

Comment:
do you think we should name annotation to imply CAO?

Reply:
I don't think there's a need for that now, and this is not an API so we can change it anytime in the future.

nit: you could just `deployment.Spec.Template.Annotations["operator.openshift.io/bootstrap-user-exists"] = fmt.Sprintf("%v", bootstrapUserExists)` to avoid conditions

Reply:
Why is it desirable to avoid conditions? I think the absence of an annotation is as good a signal as one with a value of `"false"`.